### PR TITLE
User can change language

### PR DESCRIPTION
**Pull request by koggs**
### What we did:
- Add locales to application
- Add se.yml and se.devise.yml
- Add rails i18 gem
- Add set locale method in application controller

### CODE REVIEW

File: app/views/articles/show.html.haml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 - @article.comments.each do |comment|
   %p= comment.content
 
 = form_with scope: :comment, url: article_comments_path(@article), local: true do |form|
-  = form.label :content
-  = form.text_field :content
+  = form.label t('article.content')
+  = form.text_field :content, placeholder: t('article.content')
   = form.submit
```
Comment (alghazu) [inline app/views/articles/show.html.haml]: submit button missing translation

**Review by alghazu — CHANGES_REQUESTED** [review]

File: app/views/articles/show.html.haml (hunk 1 above)
Comment (koggs) [reply app/views/articles/show.html.haml]: amongst other things but it's fixed :)

**Review by alghazu — APPROVED** [review]
:+1:

File: app/views/layouts/_navbar.html.haml [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
-= link_to "Sign In", new_user_session_path
-= link_to "Sign Up", new_user_registration_path
+= link_to t('navbar.create'), new_article_path
+= link_to t('navbar.sign_in'), new_user_session_path
+= link_to t('navbar.sign_up'), new_user_registration_path
+
+- if I18n.locale != I18n.default_locale
+  = link_to "English", { locale: 'en' }
+- else
+  = link_to "Svenska", { locale: 'sv' }
```
Comment (tochman) [inline app/views/layouts/_navbar.html.haml]: Why don't you use locales for "English" and "Svenska"?  ;-)

**Comment by koggs** [comment]
@tochman  
Addressed comment

**Review by tochman — APPROVED** [review]
👍

**Pull request merged**